env/main.tf

```diff
@@ -358,7 +362,7 @@ resource "azurerm_key_vault_access_policy" "apim" {
 # Key Vault Secret
 resource "azurerm_key_vault_secret" "cosmos_conn" {
   name         = "cosmos-conn"
-  value        = azurerm_cosmosdb_account.cosmos.connection_strings[0]
+  value        = "${azurerm_cosmosdb_account.cosmos.connection_strings[0]}&retryWrites=false"
```

Comment:
Without retryWrites=false, there is below error. I think once the Cosmos DB account is enabled with server-side retry capability, we don't need this parameter.

Exception: MongoError: Retryable writes are not supported. Please disable retryable writes by specifying "retrywrites=false" in the connection string or an equivalent driver specific config.

In addition, it will be better if server side can add retryWrites parameter =true/false in the connection string based on the server-side retry capability.